.gitignore

```diff
@@ -24,3 +23,7 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 package-lock.json
+
+# Temporary for script to insert copyright
+copyrightText.txt
```

Comment:
Can you remove this here now as well? Trying to keep the files as clean as they can get

Reply:
Yup removed those, made a script to quickly add copyright headers to anything that didn't have it in the file.